app/assets/javascripts/deploy_groups.js

```diff
@@ -0,0 +1,17 @@
+// get back a list of deploy groups, generate a select list with 'em
```

Comment:
`// when user changes the deploy group, fill in the projects from data stored on the page`